server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimUserProvisioning.java

```diff
@@ -536,18 +529,11 @@ protected int deleteUser(String userId, int version, String zoneId) {
         return updated;
     }
 
+    @VisibleForTesting
```

Comment:
why not simply protected then from test class it is reachable

Reply:
fixed with commit df4592f